web/web/models/home_hub.py

```diff
@@ -49,6 +53,8 @@ def __repr__(self):
                                         backref=db.backref('service_location'),
                                         uselist=False)
 
+Market.home_hubs = relationship('HomeHub',
```

Comment:
Is having this relationship on the Market model (file) causing a circular import issue? The way it's written, intuitively, tells me that it shouldn't. Am I missing something?

Reply:
Yes ideally this relationship would be declared on models/market.py but because the market table is imported on home_hub.py to set the foreign key, it errors:
 ImportError: cannot import name 'HomeHub' from partially initialized module 'web.models.home_hub' (most likely due to a circular import) (/Users/apeery/Documents/GISMo/TESS/web/web/models/home_hub.py)

so my work around is setting this relationship on models/home_hub.py which seems to work well